cmd/ctr/namespaces.go

```diff
@@ -0,0 +1,201 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"sort"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/containerd/containerd/metadata"
+	"github.com/docker/swarmkit/log"
```

Comment:
my gofmt was doing this as well, had to just remove `github.com/docker/swarmkit/log` from my go path to get it to stop